smart_card_connector_app/src/chrome-api-jstocxxtest.js

```diff
@@ -93,6 +94,19 @@ goog.exportSymbol('testChromeApiProviderToCpp', {
   'testSmoke': async function() {
     launchPcscServer(/*initialDevices=*/[]);
     await pcscReadinessTracker.promise;
+  },
+
+  'testEstablishContext': async function() {
+    launchPcscServer(/*initialDevices=*/[]);
+    await pcscReadinessTracker.promise;
+    chrome
+        .smartCardProviderPrivate['reportEstablishContextResult'](
+            /*requestId=*/ 1,
```

Comment:
Optional, but maybe some more distinguished number like 42 or 123? So that the tests don't accidentally pass if some kind of counter is used accidentally.

Reply:
done